### PR TITLE
BasketListItem - display variation properties

### PR DESCRIPTION
**Pull request by cjohannsen**
Requires: https://github.com/plentymarkets/plugin-io/pull/371

### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/js/src/app/helper/utils.js [hunk 1]
```diff
@@ -63,3 +63,20 @@ export function defaultValue(input, defaultValue)
 
     return input;
 }
+
+export function orderArrayByKey(array, key, desc)
```
Comment (lks-mtzn) [inline resources/js/src/app/helper/utils.js:67]: Du kannst hier stattdessen orderBy von lodash nehmen.

File: resources/js/src/app/filters/variationProperties.filter.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { orderArrayByKey } from "../helper/utils";
+
+const PROPERTY_ORDER_BY_KEY = "position";
+
+Vue.filter("variationProperties", (variationPropertyGroups, variationProperties, propertyTypes = [], displaySetting) =>
+{
+    let properties = variationProperties.map(property => property.property);
+
+    if (propertyTypes.length)
```
Comment (lks-mtzn) [inline resources/js/src/app/filters/variationProperties.filter.js]: Mach eine Schleife raus. Und Filter dann.

File: resources/js/src/app/filters/variationProperties.filter.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        properties = properties.filter(property => propertyTypes.includes(property.cast));
+    }
+    if (displaySetting.length)
+    {
+        properties = properties.filter(property => property.display.includes(displaySetting));
+    }
+
+    const groups = [];
+
+    for (const group of variationPropertyGroups)
```
Comment (lks-mtzn) [inline resources/js/src/app/filters/variationProperties.filter.js]: Hier das selbe eig.

**Review by lks-mtzn — CHANGES_REQUESTED** [review]

**Comment by cjohannsen** [comment]
restart review

**Review by felixdausch — APPROVED** [review]

**Review by lks-mtzn — APPROVED** [review]

**Pull request merged**